OtripleS.Web.Api.Tests.Unit/Services/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Validations.RetrieveAll.cs

```diff
@@ -0,0 +1,50 @@
+﻿//---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+//----------------------------------------------------------------
+
+using FluentAssertions;
+using Moq;
+using OtripleS.Web.Api.Models.CalendarEntryAttachments;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace OtripleS.Web.Api.Tests.Unit.Services.CalendarEntryAttachments
+{
+    public partial class CalendarEntryAttachmentServiceTests
+    {
+        [Fact]
+        public void ShouldLogWarningOnRetrieveAllWhenCalendarEntryAttachmentsWereEmptyAndLogIt()
+        {
+            // given
+            IQueryable<CalendarEntryAttachment> emptyStorageCalendarEntryAttachments = new List<CalendarEntryAttachment>().AsQueryable();
+            IQueryable<CalendarEntryAttachment> expectedCalendarEntryAttachments = emptyStorageCalendarEntryAttachments;
+
+            this.storageBrokerMock.Setup(broker =>
+                    broker.SelectAllCalendarEntryAttachments())
+                .Returns(emptyStorageCalendarEntryAttachments);
```

Comment:
Please follow our indentation standard:

```csharp
SomeMethodCall()
    .SomeChainCall()
        .SomeOtherCall();
```

Reply:
Should I update and send a PR again or will it be taken care in the code rub?

Reply:
You should fix it in this PR.